packages/kbn-openapi-common/schemas/primitives.schema.yaml

```
@@ -8,8 +8,8 @@ components:
   schemas:
     NonEmptyString:
       type: string
-      pattern: ^(?! *$).+$
       minLength: 1
+      format: nonempty
       description: A string that is not empty and does not contain only whitespace
```

It looks like the description should be changed a bit to reflect that we handle non only spaces

```suggestion
      description: A string that is not empty and does not contain only whitespace characters
```